tools/cldr-apps/js/src/esm/cldrInfo.js

```diff
@@ -569,6 +569,26 @@ function updateRowVoteInfo(tr, theRow) {
       cldrDom.createChunk(msg, "p", "alert alert-warning fix-popover-help")
     );
   }
+  if (theRow.voteTranscript) {
+    const transcriptBox = cldrDom.createChunk(
+      "",
+      "div",
+      "transcript-container"
+    );
+    const transcriptText = cldrDom.createChunk(
+      theRow.voteTranscript,
+      "pre",
+      "transcript-text"
+    );
+    transcriptBox.appendChild(transcriptText);
+    const transcriptNote = cldrDom.createChunk(
+      cldrText.get("transcript_note"),
+      "p",
+      "alert alert-warning"
+    );
+    transcriptBox.appendChild(transcriptNote);
+    tr.voteDiv.appendChild(transcriptBox);
+  }
```

Comment:
soon to be a vue component?

Reply:
but as i mentioned this is a preview…  there may be ways to visualize the data better, however, i want to make sure i'm getting the right data first.